src/main/target/common_pre.h

```diff
@@ -175,6 +175,10 @@
 #define USE_SERIALRX_SPEKTRUM   // SRXL, DSM2 and DSMX protocol
 #define USE_SERIALRX_SUMD       // Graupner Hott protocol
 
+#ifndef STM32F3 //Not enough space for F3 support
+#define USE_SERIALRX_SRXLv2     // Spektrum SRXL2 protocol
```

Comment:
This is non-standard. The `USE_` define should be all uppercase.

Reply:
Also the define should be added to the `#if (FLASH_SIZE > 256)` section rather than adding an additional F3 exclusion here.

Reply:
> This is non-standard. The `USE_` define should be all uppercase.

Understood, I've renamed all SRXLv2 references in any case to be SRXL2 ('v' removed). I think this more sense than having a capital V included with the protocol name itself, and it matches how it's named in our public documentation anyway.

Reply:
> Also the define should be added to the `#if (FLASH_SIZE > 256)` section rather than adding an additional F3 exclusion here.

Would it make sense to have a new `#if (FLASH_SIZE > 256)` section next to the other serial protocols in case any other protocols come down the line (would be nice to have the USE_SERIALRX_xxxx in the same section), or would that just cause more confusion down the line? I will include it at the very end of the current section otherwise.

Reply:
From a maintainability perspective I would prefer just adding it to the existing `#if (FLASH_SIZE > 256)` section and not create an additional area for maintenance. But let's hear some other opinions before changing.

Reply:
> From a maintainability perspective I would prefer just adding it to the existing `#if (FLASH_SIZE > 256)` section and not create an additional area for maintenance. But let's hear some other opinions before changing.

I've moved it to the larger > 256 section. I think down the line if the serial protocols want to be grouped together in a new section, it can be changed then.

Reply:
@etracer65 Can you mark this as resolved? I think this is the last change request preventing the merge. 
Thanks